ios/RNNativePlayground/RNOldHotelsOptions.h

```diff
@@ -0,0 +1,8 @@
+#import <UIKit/UIKit.h>
+#import <RNKiwiMobile/RNKiwiMobile.h>
+
+@interface RNOldHotelsOptions : UIViewController <RNKiwiOptions>
```

Comment:
I would keep everywhere same naming (like in JS). So, `RNHotels...` and `RNNewHotels...`, without the `Old` anywhere.

Reply:
Refactored